src/EA.Weee.Web/Areas/Aatf/Views/EvidenceTonnageValueCopyPaste/Index.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@using EA.Prsd.Core.Web.Mvc.RazorHelpers
+@using EA.Weee.Web.Areas.Aatf.Controllers
+@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@model EA.Weee.Web.Areas.Aatf.ViewModels.EvidenceTonnageValueCopyPasteViewModel
+
+@section full_main_content
+{
+    @(this.WeeeGds().BackLink(@Url.UrlFor<EvidenceTonnageValueCopyPasteController>(a => a.Index(Model.OrganisationId, Model.Action, true))))
+    <header class="hgroup">
+        <div>
+            <h1 class="govuk-heading-l">WEEE received for treatment</h1>
+        </div>
+    </header>
+    <p class="govuk-body">
+        Provide tonnage to 3 decimal places for 14 categories. For example 1.238
+    </p>
+    <p class="govuk-body">
+        You can copy the WEEE in tonnes from an Excel spreadsheet
+    </p>
+    <p class="govuk-body">
+        Data entered here will be interpreted as category 1 at the top through to category 14 at the bottom
+    </p>
+    @using (Html.BeginForm("Index", "EvidenceTonnageValueCopyPaste", FormMethod.Post))
+    {
+        @Html.AntiForgeryToken()
+        @Html.Gds().ValidationSummary()
+        @Html.HiddenFor(m => m.OrganisationId)
+        @Html.HiddenFor(m => m.AatfId)
+        @Html.HiddenFor(m => m.EvidenceId)
+        @Html.HiddenFor(m => m.Action)
+
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-one-half">
+                <h2 class="govuk-heading-s" id="received-entry">Total received (tonnes)</h2>
+                <textarea class="govuk-textarea govuk-!-width-three-quarters" placeholder="Paste values here" id="ReceievedPastedValues" name="ReceievedPastedValues" style="height: 400px;" aria-labelledby="received-entry"></textarea>
+            </div>
+            <div class="govuk-grid-column-one-half">
+                <h2 class="govuk-heading-s" id="reused-entry">Reused as whole appliances (tonnes)</h2>
+                <textarea class="govuk-textarea govuk-!-width-three-quarters" placeholder="Paste values here" id="ReusedPastedValues" name="ReusedPastedValues" style="height: 400px;" aria-labelledby="reused-entry"></textarea>
+            </div>
+        </div>
+        <button type="submit" class="govuk-button" id="aatf-task-list" data-module="govuk-button" data-prevent-double-click="true">
```

Comment:
can you use @Html.Gds() control for this button?

Reply:
Done